packages/block-editor/src/components/block-parent-selector/index.js

```diff
@@ -54,13 +76,15 @@ export default function BlockParentSelector() {
 			<div
 				className="block-editor-block-parent-selector"
 				key={ firstParentClientId }
+				ref={ nodeRef }
+				{ ...showMoversGestures }
 			>
 				<ToolbarButton
 					className="block-editor-block-parent-selector__button"
 					onClick={ () => selectBlock( firstParentClientId ) }
 					label={ sprintf(
 						/* translators: %s: Name of the block's parent. */
-						__( 'Select parent (%s)' ),
+						__( 'Select %s' ),
```

Comment:
In #26135 this change was also proposed, but I don't think it's a good idea for situations where a block might be nested in same block type - e.g. a group within a group. That seems very confusing, I think it's probably quite confusing already with the icon being the same as the block icon.

Are there other options? I proposed just making it 'Select parent'. Alternatively could the text wrap in the text labels mode?

cc @tellthemachines.

Reply:
I think the same block nested within itself is enough of a special case that we might just as well treat that one separately. "Select parent" is rather technical and cumbersome for most cases (for example, selecting the Gallery that contains an Image, or the Quote that has a paragraph, etc). I think all of these cases benefit from a more straightforward phrasing.